slides/04-networking-in-riot/networking-in-riot.md

```diff
@@ -450,6 +464,40 @@ Follow the instructions in the notebook **riot/networking/coap/coap.ipynb**
 
 ---
 
+## Bonus: BLE
+
+**NimBLE**: Open-source Bluetooth 5.1 stack (both Host & Controller) for Nordic chipsets.
+
+Features highlight:
+- Support for 251 byte packet size6
+- Support for all 4 roles concurrently - Broadcaster, Observer, Peripheral and Central
+- Support for up to 32 simultaneous connections
+- Legacy and SC (secure connections) SMP support (pairing and bonding)
+- Advertising Extensions
+- Bluetooth Mesh
+- Eddystone beacon protocol (Eddystone-UID & Eddystone-URL only)
+- **Low Level API access**
+
+.center[
+    <img src="images/ble.png" alt="" style="width:300px;"/>
+    <img src="images/mynewt.png" alt="" style="width:150px;"/>
+]
+
+---
+
+## Exercise: BLE Nimble scan-advertise
+
+Follow the instructions in the notebook **riot/ble/ble_scan/ble_scan.ipynb**
+
+.center[
+<form class=notebook>
+    <input class=login id="login_ble" type="text" oninput="check_login('login_ble', 'launcher_ble')" placeholder="Enter your IoT-LAB login">
+    <input class=launcher id="launcher_ble" type="button" value="Launch notebook" onclick="open_notebook('login_ble', 'riot/ble/ble_scan/ble_scan.ipynb')" disabled>
+</form>
+]
+
+---
+
```

Comment:
@emmanuelsearch, here I should add mention that it can run with IP, RPL, architecture and build system

Reply:
@emmanuelsearch here is the rendering:

![image](https://user-images.githubusercontent.com/23060007/122400662-32cb2300-cf7c-11eb-8cf2-b2324dbf7a42.png)